js/cssEditor.js

```diff
@@ -0,0 +1,10 @@
+function starter(){
+    //My variables selecting the necessary text boxes
+let htmlText = document.getElementById("htmlText");
+let cssText = document.getElementById("cssText");
+let browserF = document.getElementById("browserFrame");
+let bfcw = browserF.contentWindow.document;
```

Comment:
`const` would be a great choice for us except `let`. Because we are not updating the values.